internal/subshell/bash/bash.go

```diff
@@ -64,6 +65,15 @@ func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string
 		return errs.Wrap(err, "RcFile failure")
 	}
 
+	if path, pathExists := env["PATH"]; pathExists && runtime.GOOS == "windows" {
+		// Either the State Tool is being installed on Windows, or a runtime is being deployed on Windows.
```

Comment:
```suggestion
```

Nit: This feels oddly opinionated for a package that should not care about what consumers use it for.